src/client/components/UserHeader.less

```diff
@@ -68,6 +69,11 @@
   }
 }
 
+.UserHeader__followsyou {
+  margin-top: -30px;
```

Comment:
I'm not a fan of large margin and positioning elements like this, I prefer using flex box but I think that would involve restructuring html elements around here and there, so if @Sekhmet thinks its fine then it's fine with me

Reply:
Will wait for feedback.